src/intrinsics.cpp

```diff
@@ -259,6 +259,11 @@ static const Type *lLLVMTypeToISPCType(const llvm::Type *t, bool intAsUnsigned)
         return PointerType::GetUniform(AtomicType::VaryingDouble);
     }
 
+    // vector of pointers
+    else if (t == LLVMTypes::PtrVectorType) {
```

Comment:
Could you add `(t == LLVMTypes::PtrType)` condition here as well?

Reply:
Pointer to values are already covered on lines 233-264